fiftyone/utils/lerobot.py

```
@@ -0,0 +1,1549 @@
+
+
+def _episode_time_range(row, videos, fps):
+    if videos:
+        starts = [video.timestamps.from_timestamp for video in videos]
+        ends = [video.timestamps.to_timestamp for video in videos]
+        return (min(starts), max(ends))
+
+    return (0.0, row["length"] / fps)
+
+
+def _format_source_path(template, **coordinates):
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `_format_source_path`

(ANN202)